embedded-graphics/src/geometry/mod.rs

```diff
@@ -0,0 +1,17 @@
+//! Geometry module.
+
+mod point;
+mod size;
+
+pub use point::Point;
+pub use size::Size;
+
+/// Creates a new `Point`.
+pub const fn point(x: i32, y: i32) -> Point {
```

Comment:
Hm, doesn't look like these `point()` and `size()` fns are really necessary now. Sure they're a bit shorter than calling `Point::new()`, but it confuses the API surface when there are two ways to create a `Point`. It made a bit more sense when they were macros and the name `UnsignedCoord` is a lot longer, but now, probably not. It also confuses between `Point(10, 20)` (where `Point` is a tuple struct) and `point(10, 20)` (this function). Thoughts?

Reply:
The main reason I added these was to replicate the old macros, but I don't really like them myself. I'll remove them.